_includes/v4/examples/analysis/analysis-sample173.txt

```diff
@@ -25,7 +20,7 @@
       <tuplet copyof="#analysis.tup1" xml:id="analysis.tup4"/>
     </layer>
   </staff>
-  <staff n="3">
+  <staff n="2">

```

Comment:
Since you removed the first mrest layer, would it make sense to remove the last one, too?

Reply:
yes, makes perfect sense. The idea was to simplify the example to show only what's relevant here. Resolved in a3789930814065eb98d396cfc0278cf7bad8d9b2